<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="Widget3D" name="widget3D">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>521</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuACtion1">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuUpdate_Method">
     <property name="title">
      <string>Update Method</string>
     </property>
     <addaction name="actionCPU"/>
     <addaction name="actionGPU"/>
    </widget>
    <addaction name="menuUpdate_Method"/>
   </widget>
   <addaction name="menuACtion1"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAction1_sre_2">
   <property name="text">
    <string>Action1 sre</string>
   </property>
  </action>
  <action name="actionCPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CPU</string>
   </property>
  </action>
  <action name="actionGPU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GPU OpenCL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
  <customwidget>
   <class>Widget3D</class>
   <extends>QOpenGLWidget</extends>
   <header>widget3d.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
